openapi-spec/swagger.yaml

```diff
@@ -2462,6 +2523,61 @@ components:
                 snapshotId: string
                 snapshotName: string
                 lvPath: string
+    FileShareSnapshotUpdateSpec:
+      description: Snapshot is a description of fileshare snapshot resource.
+      allOf:
+      - $ref: '#/components/schemas/BaseModel'
+      - type: object
+        properties:
+          fileshareId:
+            type: string
+          tenantId:
+            type: string
+            readOnly: true
+          userId:
+            type: string
+            readOnly: true
+          name:
+            minLength: 1
+            maxLength: 255
+            type: string
+          description:
+            type: string
+    FileShareSnapshotRespSpec:
+      description: Snapshot is a description of fileshare snapshot resource.
+      allOf:
+      - $ref: '#/components/schemas/BaseModel'
+      - type: object
+        required:
+        - name
+        - fileshareId
+        properties:
+          fileshareId:
+            type: string
+          tenantId:
+            type: string
+            readOnly: true
+          userId:
+            type: string
+            readOnly: true
+          name:
+            type: string
+          description:
+            type: string
+          status:
+            type: string
+            readOnly: true
+          profileId:
```

Comment:
is profileId required in fileshare snapshot response?

Reply:
Yes